src/main/java/it/smartcommunitylab/aac/oidc/provider/OIDCAuthenticationProvider.java

```
@@ -192,7 +192,7 @@ public void setCustomMappingFunction(String customMappingFunction) {
     }
 
     @Override
-    public Authentication doAuthenticate(Authentication authentication) throws AuthenticationException {
```

why adding logging results in swallowing exceptions?